contracts/src/Rollup.sol

```diff
@@ -314,22 +314,21 @@ contract Rollup is RollupBase {
     {
         uint256 defenderAssertionID = assertionIDs[0];
         uint256 parentID = assertions[defenderAssertionID].parent;
-        {
-            uint256 challengerAssertionID = assertionIDs[1];
-            // Require IDs ordered and in-range.
-            if (defenderAssertionID >= challengerAssertionID) {
-                revert WrongOrder();
-            }
-            if (challengerAssertionID > lastCreatedAssertionID) {
-                revert UnproposedAssertion();
-            }
-            if (lastConfirmedAssertionID >= defenderAssertionID) {
-                revert AssertionAlreadyResolved();
-            }
-            // Require that players have attested to sibling assertions.
-            if (parentID != assertions[challengerAssertionID].parent) {
-                revert NotSiblings();
-            }
+        uint256 challengerAssertionID = assertionIDs[1];
+
+        // Require IDs ordered and in-range.
+        if (defenderAssertionID >= challengerAssertionID) {
+            revert WrongOrder();
+        }
+        if (challengerAssertionID > lastCreatedAssertionID) {
+            revert UnproposedAssertion();
+        }
+        if (lastConfirmedAssertionID >= defenderAssertionID) {
+            revert AssertionAlreadyResolved();
+        }
+        // Require that players have attested to sibling assertions.
+        if (parentID != assertions[challengerAssertionID].parent) {
+            revert NotSiblings();
```

Comment:
the scope definition here was made to get around another EVM quirk: there are limitations on the structure of the stack https://soliditydeveloper.com/stacktoodeep

this change is fine as long as it compiles